packages/babel-preset-udemy-website/tests/result/subclassing-jsx-decorators-initializers.js

```diff
@@ -144,15 +145,18 @@ function (_Component) {
   function Class() {
     var _babelHelpers$getProt;
 
-    var _temp, _this;
+    var _this;
 
     babelHelpers.classCallCheck(this, Class);
 
     for (var _len = arguments.length, args = new Array(_len), _key = 0; _key < _len; _key++) {
       args[_key] = arguments[_key];
     }
 
-    return babelHelpers.possibleConstructorReturn(_this, (_temp = _this = babelHelpers.possibleConstructorReturn(this, (_babelHelpers$getProt = babelHelpers.getPrototypeOf(Class)).call.apply(_babelHelpers$getProt, [this].concat(args))), babelHelpers.initializerDefineProperty(_this, "prop1", _descriptor, babelHelpers.assertThisInitialized(babelHelpers.assertThisInitialized(_this))), _this.prop2 = 14, _temp));
+    _this = babelHelpers.possibleConstructorReturn(this, (_babelHelpers$getProt = babelHelpers.getPrototypeOf(Class)).call.apply(_babelHelpers$getProt, [this].concat(args)));
```

Comment:
I am not sure what to make of this change. It looks innocent enough.

Reply:
Yeah, when you upgrade Babel, things like this change. You can glance at it, but you don't have to worry too much.